Wright_Will/Assignments/login_and _registration/templates/index.html

```diff
@@ -3,9 +3,45 @@
     <head lang="en">
         <meta charset="utf-8">
         <title>login</title>
+        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
         <link rel="stylesheet" href={{url_for('static',filename='styles.css')}} >
     </head>
     <body>
-        login/registration
+        <div id="wrapper">
+        {%if not user_id in session%}
+        <h2 class="heading">Login</h2>
+        <form class="log_reg" action="/process" method="post">
+            <input type="hidden" name="type" value="register">
+            <h5 class="lcol">first name:</h5>
+            <input type="text" name="fname" placeholder="first name" value="">
+            <h5 class="lcol">last name:</h5>
+            <input type="text" name="lname" placeholder="last name" value="">
+            <h5 class="lcol">email:</h5>
+            <input type="text" name="email" placeholder="email" value="">
+            <h5 class="lcol">password:</h5>
+            <input type="text" name="password" placeholder="password" value="">
+            <h5 class="lcol">confirm password:</h5>
+            <input type="text" name="confirm_pass" placeholder="password" value="">
+            <input type="submit" name="name" value="Register">
+            {% with messages = get_flashed_messages() %}
```

Comment:
Better design to have errors flash on the top of the page, particularly with long forms.